### ISSUE TITLE
TypeError when updating mapping data

### ISSUE THREAD

**Issue by andreroggeri** [wiremock/python-wiremock#27]
I'm trying to update some properties from a Mapping instance before creating

When I do this a TypeError exception is thrown

```python
mapping = Mapping.from_dict({})
mapping.status = 500 # This line fails
```

The stacktrace takes me to the following line (But the code from pip is missing the `self` parameter)

https://github.com/platinummonkey/python-wiremock/blob/52235657e8977a1c6509d94baf90c7ea503aa5d7/wiremock/base/base_entity.py#L161

**Comment by andreroggeri** [comment]
Installing from master branch fixes the problem. Can we have a new release ? :smile:

**Comment by platinummonkey** [comment]
released `2.1.2` from master 👍
